src/lsptoolshost/services/projectContextService.ts

```diff
@@ -58,6 +72,20 @@ export class ProjectContextService {
         this._contextChangeEmitter.fire({ uri, context });
     }
 
+    private async getVirtualCSharpUri(uri: vscode.Uri): Promise<vscode.Uri | undefined> {
+        const response = await vscode.commands.executeCommand<ProvideDynamicFileResponse>(
```

Comment:
@davidwengier I'm not sure if there will be any problem using this endpoint or if there's something else that should be used to find the dynamic file URI

Reply:
I don't think it would cause any issues. In theory it could trigger Razor activation, but if this only gets called when a Razor document is open in the editor, that either has happened, or is about to anyway.

Reply:
I think it would get added to the document list with a wrong uri and either be fixed or show up in misc files